src/scripts/main.js

```diff
@@ -1 +1,71 @@
 'use strict';
+
+const logo = document.querySelector('.logo');
+const page = document.querySelector('body');
+
+const firstPromise = new Promise((resolve, reject) => {
+  logo.addEventListener('click', (clickEvent) => {
+    if (clickEvent.button === 0) {
+      resolve('First promise was resolved');
+    }
+  });
```

Comment:
resolved with a message First promise was resolved on a left click in the `document`